bin/egg-init.js

```diff
@@ -254,11 +253,10 @@ function copyTo(src, dest, vars) {
 function* getRegistryUrl() {
   let url = 'https://registry.npmjs.org';
   try {
-    const ip = yield publicIp.v4();
-    const geo = geoip.lookup(ip);
-    if (geo.country === 'CN') {
-      url = 'https://registry.npm.taobao.org';
-    }
+    url = execSync('npm config get registry', {
```

Comment:
many people won't set this config, so it will cause many chinese user slow,  I will try to fix this later today.